docs/_posts/murat-gunay/2021-11-29-ner_deid_subentity_augmented_i2b2_en.md

```diff
@@ -0,0 +1,156 @@
+      .setOutputCol("ner_chunk_subentity")
+
+nlpPipeline = Pipeline(stages=[document_assembler, sentence_detector, tokenizer, word_embeddings, deid_ner, ner_converter])
+model = nlpPipeline.fit(spark.createDataFrame([[""]]).toDF("text"))
+
+results = model.transform(spark.createDataFrame(pd.DataFrame({"text": ["""A. Record date : 2093-01-13, David Hale, M.D., Name : Hendrickson, Ora MR. # 7194334 Date : 01/13/93 PCP : Oliveira, 25 years old, Record date : 1-11-2000. Cocke County Baptist Hospital. 0295 Keats Street. Phone +1 (302) 786-5227. Patient's complaints first surfaced when he started working for Brothers Coal-Mine."""]})))
+```
+```scala
+...
+val word_embeddings = WordEmbeddingsModel.pretrained("embeddings_clinical", "en", "clinical/models")
+      .setInputCols(Array("sentence", "token"))
+      .setOutputCol("embeddings")
+
+val deid_ner = MedicalNerModel.pretrained("ner_deid_subentity_augmented_i2b2", "en", "clinical/models") 
+      .setInputCols(Array("sentence", "token", "embeddings")) 
+      .setOutputCol("ner")
+
+val ner_converter = NerConverter()
+      .setInputCols(Array("sentence", "token", "ner"))
+      .setOutputCol("ner_chunk_subentity")
+
+val nlpPipeline = new Pipeline().setStages(Array(document_assembler, sentence_detector, tokenizer, word_embeddings, deid_ner, ner_converter))
+val model = nlpPipeline.fit(spark.createDataFrame([[""]]).toDF("text"))
+
+val result = pipeline.fit(Seq.empty["A. Record date : 2093-01-13, David Hale, M.D., Name : Hendrickson, Ora MR. # 7194334 Date : 01/13/93 PCP : Oliveira, 25 years old, Record date : 1-11-2000. Cocke County Baptist Hospital. 0295 Keats Street. Phone +1 (302) 786-5227. Patient's complaints first surfaced when he started working for Brothers Coal-Mine."].toDS.toDF("text")).transform(data)
+```
+</div>
+
+## Results
+
+```bash
++-----------------------------+-------------+
+|chunk                        |ner_label    |
++-----------------------------+-------------+
+|2093-01-13                   |DATE         |
+|David Hale                   |DOCTOR       |
+|Hendrickson, Ora             |PATIENT      |
+|7194334                      |MEDICALRECORD|
+|01/13/93                     |DATE         |
+|Oliveira                     |DOCTOR       |
+|25                           |AGE          |
+|1-11-2000                    |DATE         |
+|Cocke County Baptist Hospital|HOSPITAL     |
+|0295 Keats Street            |STREET       |
+|(302) 786-5227               |PHONE        |
+|Brothers Coal-Mine Corp      |ORGANIZATION |
++-----------------------------+-------------+
+```
+
+{:.model-param}
+## Model Information
+
+{:.table-model}
+|---|---|
+|Model Name:|ner_deid_subentity_augmented_i2b2|
+|Compatibility:|Spark NLP for Healthcare 3.3.2+|
+|License:|Licensed|
+|Edition:|Official|
+|Input Labels:|[sentence, token, embeddings]|
+|Output Labels:|[ner]|
+|Language:|en|
+
+## Data Source
+
+A custom data set -  an uagmented version of 2014 i2b2 Deid dataset.
```

Comment:
In-house annotations based on `2014 i2b2 Deid dataset.`

Reply:
@josejuanmartinez 
there was no in-house annotations. we used i2b2 official data set and we just found another conll from another project and combined them together.

Reply:
@murat-gunay we can still mention `in-house annotations`  to give an impression (!)